### ISSUE TITLE
Keyboard shortcuts not working

### ISSUE THREAD

**Issue by matthewdias** [koekeishiya/kwm#64]
I can't get any of the keyboard shortcuts to work. My .kwmrc is mostly default. Is there something I'm missing?

**Comment by koekeishiya** [comment]
Make sure your kwmrc contains `kwmc config hotkeys enable` and that the `kwmc config prefix` line is commented.

Also if you had a terminal in focus when testing, check that Secure Keyboard Entry is not enabled.

**Comment by matthewdias** [comment]
That worked thanks.

**Comment by atelic** [comment]
Hi @koekeishiya, I'm experiencing the same problem. I installed `kwm` through `homebrew` and have version `4.0.2`. I am using the example config from [here](https://github.com/koekeishiya/kwm/tree/master/examples) and added `kwmc config hotkeys enable` to that and am still unable to get any shortcuts to work. `kwm` is listed in the Accessibility panel as well.  I'm running macOS 10.12.1

**Comment by chrishoage** [comment]
@atelic Keybindings were moved to their own application in version `4.0.1`. Check the release notes here: https://github.com/koekeishiya/kwm/releases/tag/v4.0.1

TLDR: If you'd like keybindings though a rc file daemon look at https://github.com/koekeishiya/khd
You can use any keybinding application you desire (for example BetterTouchTool could also work)
